openshift/deployment.yaml

```diff
@@ -0,0 +1,160 @@
+apiVersion: v1
+kind: Template
+metadata: {}
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    creationTimestamp: null
+    name: ${NAME}${SUFFIX}
+- kind: Role
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: ${NAME}${SUFFIX}
+  rules:
+  - apiGroups: [""] # "" indicates the core API group
+    resources: ["pods"]
+    verbs: ["get", "watch", "list"]
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    creationTimestamp: null
+    name: ${NAME}${SUFFIX}
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: Role
+    name: ${NAME}${SUFFIX}
+  subjects:
+  - kind: ServiceAccount
+    name: ${NAME}${SUFFIX}
+- apiVersion: apps.openshift.io/v1
+  kind: DeploymentConfig
+  metadata:
+    annotations:
+      openshift.io/generated-by: OpenShiftNewApp
+    creationTimestamp: null
+    labels:
+      app: ${NAME}
+    name: ${NAME}${SUFFIX}
+  spec:
+    replicas: 1
+    selector:
+      app: ${NAME}${SUFFIX}
+      deploymentconfig: ${NAME}${SUFFIX}
+    strategy:
+      resources: {}
+    template:
+      metadata:
+        annotations:
+          openshift.io/generated-by: OpenShiftNewApp
+        creationTimestamp: null
+        labels:
+          app: ${NAME}
+          deploymentconfig: ${NAME}${SUFFIX}
+      spec:
+        serviceAccountName: "${NAME}${SUFFIX}"
+        containers:
+        - name: ${NAME}${SUFFIX}
```

Comment:
Container names don't need to be unique within the namespace, just within the pod. You can just use "tomcat" as container name. Don't forget to update `ConfigChange` trigger a few lines below.